bbc/simorgh
# WSTEAM1-1099 file Uploader - UGC - Implement duplicate file upload logic

Resolves JIRA [WSTEAM1-1099](https://jira.dev.bbc.co.uk/browse/WSTEAM1-1099)

Overall changes
======
Adds logic to rename duplicate files to have their number after it ie filename , filename (2) so that the live region will read out when they are added and removed
 
Code changes
======

- _A bullet point list of key code changes that have been made._

Testing
======
1. Visit http://localhost.bbc.com:7081/somali/send/u130092370?renderer_env=test
2. Upload duplicate files and observe different names
3. Using a screenreader, do the same thing and listen for the live region announcing their addition and deletion

Screen readers to check:

Mobile

- [x] VoiceOver / Safari
- [x] Talkback / Chrome

Desktop

- [x]  VoiceOver / Safari: 
- [x]  Jaws / Chrome / Edge: 
- [x]  NVDA / Chrome / Firefox: 

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
